composer.json

```diff
@@ -21,12 +21,12 @@
         }
     ],
     "require": {
-        "craftcms/cms": "^4.0.0",
+        "craftcms/cms": "^4.0.0|^5.0.0",
         "php": "^8.0.2",
-        "weareferal/remote-core": "4.1.2"
+        "weareferal/remote-core": "dev-craft-5"
```

Comment:
update this after https://github.com/timmyomahony/craft-remote-core/pull/26 is merged